instagram_app/src/Components/SearchBar/SearchBar.js

```diff
@@ -0,0 +1,37 @@
+import React from 'react';
+import './SearchBar.css';
+import Button from 'react-bootstrap/Button';
+import Nav from 'react-bootstrap/Nav';
+import Navbar from 'react-bootstrap/Navbar';
+import Form from 'react-bootstrap/Form';
+import FormControl from 'react-bootstrap/FormControl';
+
+function SearchBar(props) {
+    return (  
+        <div className='search-bar'>
+            <Navbar bg="light" expand="lg">
+            <Navbar.Brand  href="#home"><img className='logo' src="https://img.icons8.com/plasticine/100/000000/instagram.png" alt='logo'></img></Navbar.Brand>
+            <img className='writing' src={require('../../Images/instalogo.png' )}  alt='writingLogo'></img>
+            <Navbar.Toggle  aria-controls="basic-navbar-nav" />
+            <Navbar.Collapse id="basic-navbar-nav">
+                <Form inline className='search , mr-auto'>
```

Comment:
if you are to make the page searchable a function will need to be written in app.js and passed down to this component. When you are practicing React, see if you can write the logic for filtering the data in dummydata by the search input that's put on state in your form....which then means this would need to be changed to a class component 😄